### PR TITLE
feat!: templates for sern v2

### PR DESCRIPTION
**Pull request by jacoobes**

### CODE REVIEW

File: templates/typescript/src/index.ts [hunk 1]
```diff
@@ -9,12 +15,36 @@ const client = new Client({
...
+			.add({ '@sern/logger': single(new DefaultLogging()) }),
+});
+
 //View docs for all options
 Sern.init({
-	client,
 	defaultPrefix: '!', // removing defaultPrefix will shut down text commands
-	commands: 'dist/commands',
-	// events: 'dist/events' (optional),
+	commands: 'src/commands',
```
Comment (EvolutionX-10) [inline templates/typescript/src/index.ts]: shouldn't it be dist?

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
has this been tested, like the exact template (all js,ts,,js-esm) properly?

File: templates/typescript/src/index.ts (hunk 1 above)
Comment (jacoobes) [reply templates/typescript/src/index.ts]: yeah

Comment (jacoobes) [reply templates/typescript/src/index.ts]: oops

**Comment by jacoobes** [comment]
https://cdn.discordapp.com/attachments/941005497803374702/1058062009633099897/image.png

**Comment by jacoobes** [comment]
one more thing to double check

**Comment by jacoobes** [comment]
> has this been tested, like the exact template (all js,ts,,js-esm) properly?

Yes, should be all good

**Review by EvolutionX-10 — APPROVED** [review]

**Comment by jacoobes** [comment]
Poggers

**Pull request merged**